wiki/Live_streaming_osu!/en.md

```diff
@@ -1,37 +1,36 @@
-If you are visiting this page, then it is likely you have an interest and want to learn how to live stream [osu!](osu! "wikilink") on twitch.tv. Doing so is a very simple process but you may require a reliable computer to be able to run both osu! and send live encoding to [twitch.tv](http://www.twitch.tv) (which from now on will be referred to as TTV) at the same time.
+# Live streaming osu!
 
-Step 1 - Registration
-=====================
+If you are visiting this page, then it is likely you have an interest and want to learn how to live stream *osu!* on twitch.tv. Doing so is a very simple process but you may require a reliable computer to be able to run both osu! and send live encoding to [twitch.tv](https://www.twitch.tv) (which from now on will be referred to as TTV) at the same time.
 
-<img src="TTV_RegistrationSteps.jpg" title="Registration Steps for Twitch.tv" alt="Registration Steps for Twitch.tv" width="200" />
+## Step 1 - Registration
 
-1.  Registering for an account for TTV is very easy, just head to the website <http://www.twitch.tv> and click on the "Sign Up" button in the top right corner.
+![Registration Steps for Twitch.tv](img/TTV_RegistrationSteps.jpg "Registration Steps for Twitch.tv")
+
+1.  Registering for an account for TTV is very easy, just head to the website https://www.twitch.tv and click on the `Sign Up` button in the top right corner.
 2.  Afterwards, fill in the fields.
 3.  You will get an activation link via e-mail.
-4.  Once you're done, your channel will be at www.twitch.tv/"username" (username being your chosen username).
+4.  Once you are done, your channel will be at www.twitch.tv/"username" (username being your chosen username).
```

Comment:
change `www.twitch.tv/"username"` -> `` `www.twitch.tv/{username}` `` (and make sure it is in code)